src/main/java/io/netty/incubator/codec/quic/QuicheQuicStreamChannel.java

```diff
@@ -367,6 +369,63 @@ public int compareTo(Channel o) {
         return id.compareTo(o.id());
     }
 
+    /**
+     * Returns the ID of this channel.
+     */
+    @Override
+    public int hashCode() {
+        return id.hashCode();
+    }
+
+    /**
+     * Returns {@code true} if and only if the specified object is identical
+     * with this channel (i.e: {@code this == o}).
+     */
+    @Override
+    public boolean equals(Object o) {
+        return this == o;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Since remoteAddress() and localAddress() both return this.address, the code could be simplified. No need to print both.